### PR TITLE
Append common actions to the PandasClient

### PR DESCRIPTION
**Pull request by tonyfast**
This pull request adds list_tables, load_data, and create_table attributes for the PandasClient.

### CODE REVIEW

**Comment by tonyfast** [comment]
Thanks for the comments.  Are these failing circle tests something that this PR caused?

**Comment by cpcloud** [comment]
@tonyfast nope, I'm working on fixing them in #1458

File: ibis/pandas/client.py [hunk 1]
```diff
@@ -321,6 +323,64 @@ def compile(self, expr, *args, **kwargs):
...
+        Parameters
+        ----------
+        name : string
+        database : string, default None
+
+        Returns
+        -------
+        if_exists : boolean
+        """
+        return name in self.list_tables()
```
Comment (cpcloud) [inline ibis/pandas/client.py]: I still think this should be `name in self.dictionary`. No reason to make this unnecessarily slow.

**Comment by xmnlab** [comment]
@tonyfast maybe you can use this workaround https://github.com/Quansight/ibis/blob/master/ci/requirements-dev-3.6.yml#L17 until #1458 is not done.

**Comment by cpcloud** [comment]
@tonyfast rebase on master and you'll be able to get past any failures that aren't related this PR.

**Comment by cpcloud** [comment]
@tonyfast Looks like there's another issue, putting a PR up now to fix it.

**Comment by cpcloud** [comment]
#1468 should fix these errors.

File: ibis/pandas/client.py [hunk 2]
```diff
@@ -321,6 +323,64 @@ def compile(self, expr, *args, **kwargs):
...
+        """
+        Parameters
+        ----------
+        table_name : string
+        obj: pandas.DataFrame
+        """
+        # kwargs is a catch all for any options required by other backends.
+        self.dictionary[table_name] = pd.DataFrame(obj)
+
+    def create_table(self, table_name, obj=None, schema=None):
```
Comment (cpcloud) [inline ibis/pandas/client.py]: Can you add a test for this?

**Comment by cpcloud** [comment]
@tonyfast it looks like you're out of date with master, can you rebase again?

**Comment by tonyfast** [comment]
👌Thanks @cpcloud!

**Comment by cpcloud** [comment]
thanks @tonyfast!

**Pull request closed without merging**